ffmpeg/_run.py

```diff
@@ -118,10 +118,11 @@ def get_args(stream_spec, overwrite_output=False):
     input_nodes = [node for node in sorted_nodes if isinstance(node, InputNode)]
     output_nodes = [node for node in sorted_nodes if isinstance(node, OutputNode)]
     global_nodes = [node for node in sorted_nodes if isinstance(node, GlobalNode)]
-    filter_nodes = [node for node in sorted_nodes if isinstance(node, FilterNode)]
+    filter_nodes = [node for node in sorted_nodes if isinstance(node, (FilterNode, SourceNode))]
```

Comment:
How about we just check if it's a `FilterableNode` instead of checking for both implementations?

Reply:
IDK why I didn't think of that. Anyway, to obtain the inputs using the libavfilter syntax I pointed out in the stackoverflow link, we would need to do some more checks anyway.